web/util.ts

```diff
@@ -117,8 +117,23 @@ export function createElement(options: any): any {
     return element;
 }
 
-export function toPaddedHex(x: number) {
-    let octet = x.toString(16);
-    if (octet.length < 2) octet = "0" + octet;
-    return octet;
-}
\ No newline at end of file
+/*
+export class Rgb {
```

Comment:
I have a `Color` class lying around from another project that is chock-frickin'-full of helpful color-related functions. I would consider bringing that class into this project, or adding a new npm package dependency, if you are hoping to add more complicated color-related functionality. However, I would prefer not to merge the fully commented-out class into master. Please either complete this class (and put it in its own module), suggest a dependency, or remove it.

Here is the aforementioned Color class from another project. If it was used it would need a little cleaning up, but I think I would prefer a new module based on this one over introducing another external dependency. https://github.com/pineapplemachine/jsux/blob/master/color.html

Reply:
This change request still stands, please remove the commented out class and add a Color helper later when/if it is needed.